html/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version</title>
```

Comment:
It is better to change your title to "code review sample" or something that feet for this project.

Reply:
- [ ] Please try to explain to students the reason why you request changes so that they be aware and have the motivation to implement it .you got this 👍